ru/mail/chrome/messenger/messengercompose/mailComposeEditorOverlay.dtd

```diff
@@ -1,4 +1,4 @@
-<!ENTITY attachImageSource.label         "Вложить это изображение в сообщение">
+﻿<!ENTITY attachImageSource.label         "Вложить это изображение в сообщение">
```

Comment:
@yshchebet There's no change here. Please review your changes before committing the file.